From 9f6e9d0ec53a201f62012779f1e1e8416aa47963 Mon Sep 17 00:00:00 2001
From: James Zhu <James.Zhu@amd.com>
Date: Wed, 4 Oct 2017 07:29:36 -0400
Subject: [PATCH 3632/5855] drm/amdgpu: clear build warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Clear build warning: suggest explicit braces to avoid ambiguous
‘else’ [-Wparentheses]. Introduced by commits:
bc0123e4f0d8d072a3144e7662846c5221245dc3

Signed-off-by: James Zhu <James.Zhu@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/uvd_v6_0.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/uvd_v6_0.c b/drivers/gpu/drm/amd/amdgpu/uvd_v6_0.c
index c217062..60af731 100644
--- a/drivers/gpu/drm/amd/amdgpu/uvd_v6_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/uvd_v6_0.c
@@ -525,11 +525,12 @@ static int uvd_v6_0_hw_init(void *handle)
 	}
 
 done:
-	if (!r)
+	if (!r) {
 		if (uvd_v6_0_enc_support(adev))
 			DRM_INFO("UVD and UVD ENC initialized successfully.\n");
 		else
 			DRM_INFO("UVD initialized successfully.\n");
+	}
 
 	return r;
 }
-- 
2.7.4

